### ISSUE TITLE
Allow the owner of a poll to select/deselect options of participants

### ISSUE THREAD

**Issue by mike4054** [nextcloud/polls#2347]
## Describe the goal you'd like to achieve
As a poll owner, I want to be able to select or deselect options/choices of poll participants, because it helps managing the poll result. Sometimes the poll owner negotiates with the participants to find a date that is suitable. Actually only the participant can change his/her selection, so he/she has to log in again to change the selection according to the negotiation/discussion with the organizer. If the Poll owner can change the selection directly, it's easier for the users. After the poll is closed it is clearly visible what has been chosen then.

## Describe possible solutions
Create an option switch like "Allow poll owner to change participant's selections". And eventually display this information on the poll page to be transparent.
The poll owner sees the checkboxes of the users/participants editable then.

## Additional context
None

**Comment by dartcafe** [comment]
Thanks for the good user story. Since this will he an expensive change, I'll put it into the backlog.

**Comment by lual** [comment]
(+1) I think this is very important. I did a date-poll (open for polling for 3 weeks) and and received several change requests from participants.

**Comment by dartcafe** [comment]
> (+1) I think this is very important. I did a date-poll (open for polling for 3 weeks) and and received several change requests from participants.

Why did the participants not change them by theirselves?

**Comment by lual** [comment]
> > (+1) I think this is very important. I did a date-poll (open for polling for 3 weeks) and and received several change requests from participants.
> 
> Why didn't the participants change them by theirselves?

1. Because for public participants giving an E-Mail-Address is optional. There is no possibility to edit choices without.
2. Because it's easier to call the initiator, than to find that old invite link. ;)

**Comment by lual** [comment]
This is especially a problem if you share a "Public Link" with "Options for the registration dialog" "Email address is optional". 
I did it that way. (So you can say it's my fault.)

Just to mention - there is a workaround to edit others choices: 
"Toggle Sidebar" > "Sharing" > "Copy Link to Clipboard" (from the participant you want to edit) > Open this link in an new Private/Inkognito Browser Tab

**Comment by dartcafe** [comment]
> "Toggle Sidebar" > "Sharing" > "Copy Link to Clipboard" (from the participant you want to edit) > Open this link in an new Private/Inkognito Browser Tab

Right. That is one of the reasons, why the owner has access to the private vote links. The other reason is, to be able to send this link to the participant, in case he lost or missed it. 

In the current release 3.6.0 (released right now), a cookie is set in public polls, which allows reentering the poll without the need of the private link.
